modules/assim.sequential/R/sda.enkf.R

```diff
@@ -330,6 +330,38 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL, adjustmen
     
   })
   
+  assessParams <- function(dat, Xt, mu_f_TRUE, P_f_TRUE){  
```

Comment:
You seem to have moved the definition of this function from one place within sda.enkf to a different place within sda.enkf, but haven't really explained why this function is defined within sda.enkf rather than by itself.

Reply:
Should I put it in sda.enkf.R as another function or move it to its own file?

Reply:
Generally better practice to put it in its own file. Don’t forget to export and roxygenize